profiles/kde.py

```diff
@@ -22,15 +22,11 @@ def _prep_function(*args, **kwargs):
 
 def _post_install(*args, **kwargs):
 	if "nvidia" in _gfx_driver_packages:
-		print("Plasma wayland is currently in a buggy state on Nvidia cards")
-	choice = input("Kde plasma has a wayland support would you like to install the required binaries [Y/n] ").lower()
+		print("Plasma wayland is currently in an unusable state on Nvidia cards")
```

Comment:
Can we make this "Plasma Wayland has known compatibility issues with the proprietary Nvidia driver."

Reply:
It's not unusable according to the upstream devs.

Reply:
sure, Its just in my experience as a nvidia user. its unusable

Reply:
I understand that, but the docs claim it works https://community.kde.org/Plasma/Wayland/Nvidia and it's going to get better over time, so not having to update the text is a plus. Not to mention 'in an unusable state' is very harsh wording - I'd hesitate to say that about the KDE team's work. Wayland is definitely where future development is happening, so I want the install text to have the future state of the driver + KDE in mind as well.